framework_tvb/tvb/core/adapters/abcuploader.py

```diff
@@ -100,79 +92,19 @@ def _prelaunch(self, operation, view_model, available_disk_space=0):
         if view_model.encrypted_aes_key is not None:
             trait_upload_field_names = list(self.get_form_class().get_upload_information().keys())
 
+            if TvbProfile.current.UPLOAD_KEY_PATH is None or not os.path.exists(TvbProfile.current.UPLOAD_KEY_PATH):
+                raise LaunchException("We can not process Encrypted files at this moment, "
+                                      "due to missing PK for decryption! Please contact the administrator!")
```

Comment:
Maybe leave just: "TVB can not process Encrypted files at this moment! Please contact the administrator!"